scripts/generate-versions-test-workflow.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
```

Comment:
If a GitHub Actions matrix is used, this script will no longer be needed.

Reply:
do you know how we can get all tags into the matrix?